src/modules/esl-utils/async/raf.ts

```diff
@@ -26,3 +26,6 @@ export const rafDecorator = <T extends AnyToVoidFnSignature>(fn: T): T => {
     lastArgs = args;
   } as T;
 };
+
+/** @returns promise that will be resolved after next render */
+export const promisifyNextRender = (): Promise<void> => new Promise((resolve) => afterNextRender(resolve));
```

Comment:
Open question: placement
 - 👍🏻  here `esl-utils/async/raf`
 - 🚀 `esl-utils/async/promise`

Reply:
Resolved via second way, due to purpose of method and (and in bounds of) structural changes https://github.com/exadel-inc/esl/pull/1691/commits/f39a844f3614ee703def43fb353f7c6652ffca50